projects/RabbitMQ.Client/client/impl/BasicDeliver.cs

```diff
@@ -18,12 +18,12 @@ sealed class BasicDeliver : Work
         readonly IBasicProperties _basicProperties;
         readonly ReadOnlyMemory<byte> _body;
 
-        public BasicDeliver(IBasicConsumer consumer, 
```

Comment:
Happy to roll back if you want to keep the white space

Reply:
Nah, turning down a PR because of a single whitespace change sounds unreasonable :)